README.md

```diff
@@ -10,3 +10,57 @@ single location. The goals of `issue-wanted` are to make it easier for
 programmers of all skill levels to find Haskell projects to contribute to,
 increase the number of contributions to open-source Haskell projects, and
 encourage more programmers to become a part of the Haskell community.
+
+# Testing
+
+## Prerequisites (what you need to have locally)
+
+Coming Soon
+
+## How to build
+
+To build the project, open up a terminal in the base folder and run `stack build`.
+
+## How to run server
+
+For testing the `issue-wanted` server follow these steps:
+
+1. Open up the terminal in the base folder and run `stack build`
+
+2. Run the command `stack exec issue-wanted`
+
+The server will begin running at `http://localhost:8080/`.
+
+Issue-wanted endpoints available:
+
+* `http://localhost:8080/issues`
+
+  Returns all issues.
+
+* `http://localhost:8080/issues/:id`
+
+  Returns a single issue with the corresponding id.
+
+* `http://localhost:8080/issues?label=<label name>`
+
+  Returns a list of issues with the corresponding label.  
+
+Tip: If you stop running the server and then try to restart it, you may need to run the command `fuser -n tcp -k 8080` to free the port.
+
+## How to test DB query functions
```

Comment:
I think that section can be removed from now. Testing DB queries from GHCi is more like a developer workflow. We can have a separate section like `## For Developers`, but since PR with GHCi commands is not merged yet, let's not write about this. Also because something might change here.